GIBSDownloader/tile_utils.py

```diff
@@ -78,19 +93,35 @@ def process_doubles_tuple(t_width, t_height, inter_dir, img_format, double_inter
 
     # Sequentially generate tiles
     for _, _, x, y, done_x, done_y, path in double_inter_imgs:
-        TileUtils.generate_tile_between_two_images(t_width, t_height, inter_metadata_left.end_x - inter_metadata_left.start_x, inter_metadata_left.end_y - inter_metadata_left.start_y, x, y, done_x, done_y, x - inter_metadata_left.start_x, y - inter_metadata_left.start_y, path, img_format, img_arr_left=img_arr_left, img_arr_right=img_arr_right)
+        TileUtils.generate_tile_between_two_images(t_width, t_height, inter_metadata_left.end_x - inter_metadata_left.start_x, inter_metadata_left.end_y - inter_metadata_left.start_y, x, y, done_x, done_y, x - inter_metadata_left.start_x, y - inter_metadata_left.start_y, path, img_format, img_arr_left, img_arr_right)
     
     # Close the images
     src_left.close()
     src_right.close()
     return 1
 
 def process_doubles_MP(args):
-    """ Wrapper function to unpack args """
+    """Wrapper function to unpack args for process_doubles_tuple"""
     (metadata, double_inter_imgs) = args
     return process_doubles_tuple(*metadata, double_inter_imgs)
 
 def process_quads_tuple(t_width, t_height, inter_dir, img_format, quad_inter_imgs):
+    """
+    Processes the four intermediate source images which share tiles for tiling.
+
+    This function opens the four images and calls a helper function to create the
+    tile. 
+
+    Parameters:
+        t_width (int): tile width
+        t_height (int): tile height
+        inter_dir (string): path to intermediate images directory
+        img_format (string): product image format
+        quad_inter_imgs (list): list structure containing data required for tile creation
+    """
+
+    # Get the names of intermediates - note that all entries in quad_inter_imgs
+    # are for tiles shared by the same 4 files.
     filename_TL = quad_inter_imgs[0][0]
```

Comment:
Recommend making `tl`, `bl`, etc. lowercase to remain consistent with other files.